### DISCUSSION TITLE
Favicon

### DISCUSSION THREAD

**Discussion by evripidis** [aaronwmorris/indi-allsky#1041] [Q&A] [answered]
Hello!

Quick and probably a little dump question: 
Can we add a favicon for the webpage? If yes, where it should be saved? (path).

**Comment by aaronwmorris** [comment]
I will add the code for it.

**Comment by aaronwmorris** [comment]
In #1043 I added a link for the favicon.ico at the base allsky folder under the document root.  Normally at `/var/www/html/allsky/`

**Reply by aaronwmorris** [reply]
Hold for a bit more.  I made some mistakes and I need to fix this.

**Comment by aaronwmorris** [comment, marked as answer]
Merged #1044 which includes a new logo and favicons.  Let me know what you think.

**Reply by evripidis** [reply]
Yeah, seems great 👍🏻
